packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py

```diff
@@ -121,6 +121,10 @@ class Config:
         }
 
 
+class NodeOutputs(InputSchemaWithoutCameCase):
+    outputs: dict[str, Any]
```

Comment:
Minor: If there is some structure to this dictionary it would be nice to use type hints

Reply:
I removed this code for another PR, hence resolving this conversation